### ISSUE TITLE
it seems that the decoder of the auto-encoder network is not trained in the code

### ISSUE THREAD

**Issue by czh0001** [sconlyshootery/FeatDepth#72]
Hello, it seems that the decoder of the auto-encoder network is not trained in the training code. And the image reconstruction loss, discriminative loss and convergent loss of single-view reconstruction is not included in the training loss. I think the supervision signal of auto-encoder network's encoder now only comes from the Cross-view reconstruction. Am I right?

**Comment by czh0001** [comment]
Oh, I'm sorry. They are set in cfg_kitti_fm_joint.py and I just paid attention to cfg_kitti_fm.py at first.
